rfcs/0017-grpcroute-support.md

```
@@ -638,6 +636,62 @@ when:
+|-------------|------------------|----------------|
+| 401 Unauthorized | `UNAUTHENTICATED` (16) | AuthPolicy authentication failure |
+| 403 Forbidden | `PERMISSION_DENIED` (7) | AuthPolicy authorization denial |
+| 429 Too Many Requests | `UNAVAILABLE` (14) | RateLimitPolicy rate limit exceeded |
+
+**Note:** While `RESOURCE_EXHAUSTED` (8) would be semantically more appropriate for rate limiting, the standard HTTP-to-gRPC mapping uses `UNAVAILABLE` for HTTP 429.
+
+**AuthPolicy rejection example:**
+
+When Authorino rejects a request due to missing or invalid credentials:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)